inflation/main.py

```diff
@@ -0,0 +1,22 @@
+from fastapi import FastAPI
+import logging
+from inflation.dataset.crawl import Crawler
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+app = FastAPI()
+crawler = Crawler()
+
+
+def fetch_inflation_data():
+    url = "https://docs.google.com/spreadsheets/d/1mZSqW_X_KuQQdGhH7oAn3-MATyfePLlYYmOgE-mB_9Q/edit#gid=0"
+    records = crawler.parse_excel_to_link_dataset(url)
+    crawler.crawl(records, "/inflation-resources/data/")
+
+
+@app.get("/crawl", tags=["crawl"])
+async def fetch_data():
```

Comment:
let's give this a url with default value. default value can be the one you gave already in line 14. this way we can make it parametric, if we want to have another spreadsheet later on.